core/sqf/hbase_utilities/src/main/java/org/trafodion/utility/backuprestore/TrafExportSnapshot.java

```diff
@@ -114,7 +119,10 @@
   private static final String INPUT_FOLDER_PREFIX = "export-files.";
 
   // Export Map-Reduce Counters, to keep track of the progress
-  public enum Counter { MISSING_FILES, COPY_FAILED, BYTES_EXPECTED, BYTES_COPIED, FILES_COPIED };
+  public enum Counter {
```

Comment:
Should this use some enum imported from another class? org.apache.hadoop.hbase.snapshot.ExportSnapshot.Counter?